Jenkinsfile

```
@@ -70,14 +71,15 @@ createDslContainers podName: dslPodName,
             )
           }
         }
-        else {
-          sh 'echo no more than 0'
-        }
+      } catch (Exception e) {
```

Why not put the cleanup in a finally block?

---

@ifireball what is the value in switching catch with finally?

---

`catch` is for handling failure scenarios, you need to run cleanup wither you fail or not, so `finally` is more suitable, also this avoids the need to re-throw the error and make the traceback point to the original error and not the point where it was re-thrown.